pkg/network/network.go

```diff
@@ -97,6 +97,9 @@ const (
 	// user-agent.  So we augment the probes with this header.
 	KubeletProbeHeaderName = "K-Kubelet-Probe"
 
+        // PrometheusUAPrefix defines a user agent prefix used by Prometheus. Refer to https://github.com/prometheus/prometheus/blob/master/scrape/scrape.go
+        PrometheusUAPrefix = "Prometheus/"
```

Comment:
This is used within the same package, so it should still be unexported.

Reply:
OK. I've changed that as private.